src/llm/apis/openai_completions.cpp

```diff
@@ -712,10 +708,10 @@ std::string OpenAIChatCompletionsHandler::serializeStreamingUsageChunk() {
 }
 
 void OpenAIChatCompletionsHandler::writeLogprob(Writer<StringBuffer>& writer, float logprob) {
-    // genai returns probs values which should be in the range of 0-1
+    // genai returns probs values which should be in the range of -inf-0
     // other values could be potentially invalid and should be treated as such
-    if (logprob > 0.0 && logprob <= 1.0)
-        writer.Double(std::log10(logprob));
+    if (logprob <= 0.0)
+        writer.Double(logprob);
     else
         writer.Null();
```

Comment:
Is it possible to get positive value here?

Reply:
it wouldn't be a valid value. per definition, logprobs should return negative values or 0

Reply:
In such case shouldn't we return some internal error instead of just writing null, if we get unexpected value?